routes/api.php

```diff
@@ -19,14 +20,88 @@
 |
 */
 
+/********************************** V2 ENDPOINTs ***************************************************/
+
+// login
+Route::post('/v2/manifest', [ProfileController::class, 'manifest'])->middleware(['verify-sign']);
+
+Route::middleware(['verify-sign', 'hcaptcha-verify'])->group(function () {
+    Route::post('/v2/circles', [CircleController::class, 'createCircle']);
+});
+
+/************************* TOKEN GATED endpoints *********************************/
+
+Route::prefix('v2')->middleware(['auth:sanctum'])->group(function () {
```

Comment:
ok, i think i understand now what i missed yesterday... having middleware(['auth:sanctum']) here means that all requests to routes in the group will fail if they don't have a valid token?

Reply:
correct, it checks for the middleware at the outer most layer first, sanctum will throw an Unauthenticated error if theres no valid token inside the request header.